### PR TITLE
Fortune/recording oracle

### PR DESCRIPTION
**Pull request by eugenvoronov**
## Description
Updated Recording Oracle implementation for Fortune.

## Summary of changes
Recording Oracle has been completely redesigned, adding the ability to send fortunes in bunches, as well as the ability to work with multi-networks.

## How test the changes
Request from Exchange Oracle:
**POST** `<HOST>/send-fortunes`
`[{
    "fortune": "tale",
    "workerAddress": "0x0000000000000000000000000000000000000000",
    "escrowAddress": "0x0000000000000000000000000000000000000000",
    "chainId": 80001
}]` 

## Operational checklist
- [x] [Developing Recording Oracle basic structure#160](https://github.com/humanprotocol/human-protocol/issues/160)
- [x] [Implement a local storage for temporary data#162](https://github.com/humanprotocol/human-protocol/issues/162)
- [x] [Get data from manifest#163](https://github.com/humanprotocol/human-protocol/issues/163)
- [x] [S3 client library integration#166](https://github.com/humanprotocol/human-protocol/issues/166)
- [x] [Web3 implementation of contract interfaces#167](https://github.com/humanprotocol/human-protocol/issues/167)
- [x] [Integration with Exchange Oracle#164](https://github.com/humanprotocol/human-protocol/issues/164)
- [x] [Integration with Reputation Oracle#165](https://github.com/humanprotocol/human-protocol/issues/165)
- [x] [Implement a uniqueness check algorithm#161](https://github.com/humanprotocol/human-protocol/issues/161)
- [x] [Implement checking answers for curses#171](https://github.com/humanprotocol/human-protocol/issues/171)
- [x] [Develop documentation for Recording Oracle#169](https://github.com/humanprotocol/human-protocol/issues/169)
- [x] [Multinetwork#186](https://github.com/humanprotocol/human-protocol/issues/186)
- [ ] [Service coverage with tests#168](https://github.com/humanprotocol/human-protocol/issues/168)

### CODE REVIEW

File: packages/examples/fortune/recording-oracle/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Yonatan
```
Comment (portuu3) [inline packages/examples/fortune/recording-oracle/LICENSE:3]: ??

File: packages/examples/fortune/recording-oracle/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Fortune Recording Oracle
+
```
Comment (portuu3) [inline packages/examples/fortune/recording-oracle/README.md:2]: add functional info

File: packages/examples/fortune/recording-oracle/src/routes/index.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    },
+  }, 
+}
+
+const routes: FastifyPluginAsync = async (server: FastifyInstance<Server>) => {
+  const { web3, s3, storage, escrow, curses, uniqueness } = server;
+
+  const plugins: IPlugin = { 
+    web3: {
+      [ChainId.POLYGON]: web3.create(ESCROW_NETWORKS[ChainId.POLYGON]),
```
Comment (portuu3) [inline packages/examples/fortune/recording-oracle/src/routes/index.ts:40]: could this be dynamic? like using a for each or similar

File: packages/examples/fortune/recording-oracle/tests/routes/index.test.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import server from '../../src/server';
+import { describe, test, expect } from 'vitest';
+
```
Comment (portuu3) [inline packages/examples/fortune/recording-oracle/tests/routes/index.test.ts:3]: I am missing here some test with data that does not pass validation

**Review by portuu3 — CHANGES_REQUESTED** [review]

File: packages/examples/fortune/recording-oracle/tests/routes/index.test.ts (hunk 4 above)
Comment (eugenvoronov) [reply packages/examples/fortune/recording-oracle/tests/routes/index.test.ts:3]: I will start working on tests later and take a look

**Pull request merged**